network/websocket.ksy

```diff
@@ -0,0 +1,77 @@
+meta:
```

Comment:
You haven't specified an open source license under which you license your work. Please add `meta/license` tag.

Reply:
My default is Apache 2.0, but I'm happy to license under different terms if it's easier for the maintainers.

Reply:
We are in somewhat gray legal situation here: it's not super clear if assigning a license to a .ksy file somehow transfers to results of compilation of that file. For that reason, many formats are listed as CC0, to promote maximum usage possibility in both open and closed source software: http://formats.kaitai.io/xref.html

If Apache 2.0 is your choice, then it should be probably spelled as `license: Apache-2.0`, as per https://spdx.org/licenses/Apache-2.0.html

Reply:
Changed to CC0!